core/domain/stats_domain.py

```diff
@@ -584,13 +644,36 @@ def __init__(self, answer, frequency):
         self.frequency = frequency
 
     def to_raw_type(self):
+        """Returns dict for specific answer in its raw type.
```

Comment:
Unpack 'raw type' as mentioned in previous reviews.

Reply:
Done!